Snakefile

```diff
@@ -39,3 +39,17 @@ rule cloudforest:
         "cloudforest.info"
     shell:
         "docker build -t cloudforest cloudforest/ && docker images | grep cloudforest > cloudforest.info"
+
+rule jadbio_java:
+    output:
+        "jadbio-model-exe.jar"
```

Comment:
should this be `jadbio/src/jadbio-model-exe.jar`?

Reply:
updated path as suggested